signing_service/settings/base.py

```diff
@@ -225,7 +225,7 @@
 
 # When True, it means Hawk authentication will be disabled everywhere.
 # This is mainly just to get a speed-up while testing.
-SKIP_HAWK_AUTH = False
+SKIP_HAWK_AUTH = True
```

Comment:
ok, then why did you want to skip hawk auth in the base settings? That turns off Hawk authentication by default.